packages/material-ui-lab/src/AvatarGroup/AvatarGroup.js

```diff
@@ -77,6 +88,10 @@ AvatarGroup.propTypes = {
    * Spacing between avatars.
```

Comment:
```suggestion
   * The spacing between avatars. A group with `small` spacing value has a larger overlap.
```

Reply:
Maybe we should also talk about the support for numbers in the prop description too.

Reply:
Oops, missed that. But, I'm wondering about the prop name – perhaps `overlap` would be clearer given that there's no spacing between the avatars?  Also, if I'm reading it correctly a larger number gives a smaller spacing (or overlap)? Not sure that makes sense.

It seems a demo was forgotten on the associated PR, so that could be adder here too.

Reply:
Should we add the demo to the `Avatar` component docs or to `AvatarGroup` since the `max` prop and existing demo are in the regular docs?

Reply:
I think for a couple of demos, it's fine to have them on the Avatar page. If it grows much beyond that, we can think about giving it its own page.